verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -0,0 +1,161 @@
+#include "verilog/tools/ls/symbol-table-handler.h"
+
+#include <filesystem>
+
+#include "common/strings/line_column_map.h"
+
+namespace verilog {
+
+static const std::string fileschemeprefix = "file://";
+
+bool LSPUriToPath(absl::string_view uri, std::string &path) {
```

Comment:
For writable output parametes we prefer to use pointers instead of non-const references in Verible. So `std::string *path`.